web/themes/custom/nicsdru_corplite_theme/templates/block/block--system-menu-block--main.html.twig

```diff
@@ -64,5 +64,18 @@
     {%- block content -%}
       {{- content -}}
     {% endblock %}
+
+    {# Social links, inline with the other main menu items #}
```

Comment:
I would think if social_links is set in the hse theme file, then this template should be in the hse theme rather than corp lite

Reply:
Good spot, thanks Orla. I have moved that file into the HSE template folder and reverted changes on the corp lite menu twig.